pkgs/applications/audio/nuclear/default.nix

```
@@ -0,0 +1,32 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "nuclear";
+  version = "0.6.6";
+  name = "${pname}-v${version}";
+
+  src = fetchurl {
+    url = "https://github.com/nukeop/nuclear/releases/download/v${version}/${name}.AppImage";
+    sha256 = "0c1335m76fv0wfbk07s8r6ln7zbmlqd66052gqfisakl8a1aafl6";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit name src; };
```

```suggestion
  appimageContents = appimageTools.extract { inherit name src; };
```
See https://github.com/NixOS/nixpkgs/blob/a6b361d6276b7c413ffea0a28a1a3259d2275e2e/pkgs/build-support/appimage/default.nix#L35-L37